prow/oss/plugins.yaml

```diff
@@ -137,7 +137,79 @@ plugins:
   - blunderbuss
   - lifecycle
   - override
-  
+
+  kubeflow/arena:
+  - trigger
+
+  kubeflow/batch-predict:
+  - trigger
+
+  kubeflow/caffe2-operator:
+  - trigger
+
+  kubeflow/chainer-operator:
+  - trigger
+
+  kubeflow/common:
+  - trigger
+
+  kubeflow/community-infra:
+  - trigger
+
+  kubeflow/examples:
+  - trigger
+
+  kubeflow/experimental-seldon:
+  - trigger
+
+  kubeflow/fairing:
+  - trigger
+
+  kubeflow/gcp-blueprints:
+  - trigger
+
+  kubeflow/internal-acls:
+  - trigger
+
+  kubeflow/kfp-tekton:
+  - trigger
+
+  kubeflow/kfp-tekton-backend:
+  - trigger
+
+  kubeflow/kubebench:
+  - trigger
+
+  kubeflow/kubeflow:
+  - trigger
+
+  kubeflow/metadata:
+  - trigger
+
+  kubeflow/mpi-operator:
+  - trigger
+
+  kubeflow/mxnet-operator:
+  - trigger
+
+  kubeflow/pipelines:
+  - trigger
+
+  kubeflow/reporting:
+  - trigger
+
+  kubeflow/testing:
+  - trigger
+
+  kubeflow/tf-operator:
+  - trigger
+
+  kubeflow/website:
+  - trigger
+
+  kubeflow/xgboost-operator:
+  - trigger
+
```

Comment:
```suggestion
  kubeflow/community-infra:
  - trigger

  kubeflow/gcp-blueprints:
  - trigger

  kubeflow/internal-acls:
  - trigger

  kubeflow/metadata:
  - trigger

  kubeflow/pipelines:
  - trigger

  kubeflow/testing:
  - trigger

  kubeflow/website:
  - trigger

```

Reply:
Hi @chaodaiG, going forward KF on GCP oss prow only need to support GCP sponsored projects for tests. We can leave other repos out of scope (they will be supported by a prow run by AWS).